modules/codegen-plugin/src/smithy4s/codegen/Smithy4sCodegenPlugin.scala

```diff
@@ -154,7 +155,7 @@ object Smithy4sCodegenPlugin extends AutoPlugin {
         m.root
       }
     val transforms = (conf / smithy4sModelTransformers).value
-    val s = streams.value
+    val s = (conf / streams).value
```

Comment:
The culprit - seems like the configs shared the cache (`_global` streams directory), which made Test see Compile's input hash and thus consider it different from its own. Not sure why that doesn't in turn break `Compile`'s caching (in this case it should be seeing what `Test` wrote last time it ran), but the scoping seems to resolve the issue anyway.

Reply:
Awesome find, and thank you so much for fixing !

Reply:
> Not sure why that doesn't in turn break Compile's caching (in this case it should be seeing what Test wrote last time it ran), but the scoping seems to resolve the issue anyway.

Update, it _did_ break Compile's caching apparently. So we should be all good now